content/jss_tracking_commerce.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "JSS Commerce part 4 - Tracking Cart events 🛤️"
+date = "2019-09-19"
+tags = ["XC9", "JSS"]
++++
+
+In [part 3](./jss_cart_actions.md) of our JSS and Commerce series we have been adding cart actions. It would be great if we could track these using Sitecore Experience Analytics.
```

Comment:
Misschien nog even aangeven dat dit noodzakelijk is omdat we ervoor hebben gekozen om de engine rechtstreeks te gebruiken